design/one-pager-resource-reclaim-policy.md

```diff
@@ -0,0 +1,84 @@
+
+For example, consider the following scenarios:
+
+**Kubernetes:** A `PersistentVolume` exists with a `reclaimPolicy` set to `Retain`. A `PersistentVolumeClaim` that was responsible for the dynamic provisioning of that `PersistentVolume` via reference to a `StorageClass` is deleted. The `reclaimPolicy` of the `PersistentVolume` results in the Kubernetes object being retained, and thus the external storage asset being retained.
+
+**Crossplane:** A `Managed Resource` exists with a `reclaimPolicy` set to `Retain`. A `ResourceClaim` that was responsible for the dynamic provisioning of that `Managed Resource` via reference to a `ResourceClass` is deleted. The `reclaimPolicy` of the `Managed Resource` results in the Kubernetes object being *deleted*, but the `External Resource` being retained.
+
+In short, reclaim policies in Crossplane manage the relationship between a `ResourceClaim` and the `External Resource`, while reclaim policies in Kubernetes persistent volumes manage the relationship between the `PersistentVolumeClaim` and the `PersistentVolume` Kubernetes object.
+
+## Workflow
+
+Generally, resources are provisioned via the following steps:
+
+1. A user creates a `ResourceClass`.
+
+```yaml
+apiVersion: core.crossplane.io/v1alpha1
+kind: ResourceClass
+metadata:
+  name: standard-aws-bucket
+  namespace: crossplane-system
+  annotations:
+    resource: bucket.storage.crossplane.io/v1alpha1
+parameters:
+  versioning: "false"
+  cannedACL: private
+  localPermission: ReadWrite
+  region: REGION
+provisioner: s3bucket.storage.aws.crossplane.io/v1alpha1
+providerRef:
+  name: demo-aws
+reclaimPolicy: Delete
+```
+
+2. A user creates a `ResourceClaim`.
+
+```yaml
+apiVersion: storage.crossplane.io/v1alpha1
+kind: Bucket
+metadata:
+  name: my-bucket
+  namespace: default
+spec:
+  classReference:
+    name: standard-aws-bucket
+    namespace: crossplane-system
+  name: my-bucket-1234
+```
+
+3. The creation of a `ResourceClaim` triggers a `Managed Resource` to be created using information from the claim and the referenced `ResourceClass`. If the `ResourceClass` provides a value for `reclaimPolicy` it will be set on the `Managed Resource`. If not, the `Managed Resource` will have its `reclaimPolicy` set to its default value.
+4. The `Managed Resource` provisions an `External Resource` (in this case an S3 bucket) and manages it.
+5. A user deletes the `ResourceClaim`. Because the `ConcreteResource` has an `OwnerReference` to the `ResourceClaim`, the deletion of the `ResourceClaim` triggers the deletion of the `ConcreteResource`.
+6. If the `ConcreteResource` `reclaimPolicy` is set to `Retain`, the `Managed Resource` will be deleted, but the `External Resource` will persist (i.e. the S3 bucket will still exist in your AWS account). If the `reclaimPolicy` is set to `Delete` both the `Managed Resource` and the `External Resource` will be deleted.
+
```

Comment:
This last section talks about `ConcreteResource`, which I think we renamed in the terminology section, so it's not clear what it's referring to here.

Reply:
Right! Just missed that one